e2e_test/udf/always_retry_python.slt

```diff
@@ -56,7 +56,7 @@ SELECT count(*) > 0 FROM mv_no_retry where s1 is NULL;
 t
 
 statement ok
-SET STREAMING_RATE_LIMIT=0;
+SET STREAMING_RATE_LIMIT TO "-1";
```

Comment:
Double quotes are necessary? 🤔

Reply:
Yeah if not the parser rejects it.

Reply:
```
dev=> set streaming_rate_limit=-1;
ERROR:  Failed to run the query

Caused by:
  sql parser error: Expected parameter value, found: - at line:1, column:27
Near "set streaming_rate_limit=-"
```

Reply:
```
dev=> set streaming_rate_limit to "-1";
SET_VARIABLE

```

Reply:
I think this is a flaw in the SQL parser. 

https://github.com/risingwavelabs/risingwave/blob/211b9b93ecbd18c32b3e2f9b4bc1974453a9d6ce/src/sqlparser/src/parser.rs#L3604

Should be `parse_expr` and evaluate the expression rather than `parse_value`

Reply:
Alternatively, we may support

```
SET streaming_rate_limit = UNLIMITED
```

via adding some trick here, similar to `SET timezone = LOCAL`

https://github.com/risingwavelabs/risingwave/blob/211b9b93ecbd18c32b3e2f9b4bc1974453a9d6ce/src/sqlparser/src/parser.rs#L4436

Reply:
I just do `set streaming_rate_limit = default`.